alphalens/performance.py

```diff
@@ -247,7 +247,11 @@ def quantize_factor(factor, quantiles=5, by_group=False):
     """
 
     def quantile_calc(x, quantiles):
-        return pd.qcut(x, quantiles, labels=False) + 1
+        try:
+            return pd.qcut(x, quantiles, labels=False) + 1
+        except ValueError:
+            # This is supposed to fix "ValueError: Bin edges must be unique"
+            return pd.qcut(x.rank(method='first'), quantiles, labels=False) + 1
```

Comment:
My concern is that if you are creating an alpha factor then you are implying there is meaning to the values you are assigning, whatsmore you are also implying there is meaning in the magnitude or distance between values. I'm worried that by arbitrarily throwing equities with the same values into different quantiles it would distort/skew the interpretation.